cmd/syncthing/monitor.go

```diff
@@ -181,6 +186,43 @@ func copyStderr(stderr io.ReadCloser, dst io.Writer) {
 					panicFd.WriteString(line)
 				}
 				stdoutMut.Unlock()
+
+				if optPanicReporting {
+					// Prepare query for collecting error log
+					type Report struct {
+						Error       string
+						Version     string
+						LongVersion string
+						Os          string
+						Arch        string
+					}
+					report := &Report{Error: line[:len(line)-1],
```

Comment:
This won't actually help us very much, because this line will only contain `panic: runtime error: invalid memory address or nil pointer dereference` which tells us nothing about where it happened or why. We'll need both the preceding and following lines until the child syncthing process exits, i.e. like what gets printed to the panic log... Which makes it a little bit more tricky.